### PR TITLE
feat: add annotation to set health check send proxy protocol

### PR DESCRIPTION
**Pull request by erebe**
Hello,

This PR is adding support to set if the health checks are sending proxy protocol before the actual check.
For now, there is no way to control it from Kubernetes.

See support ticket https://console.scaleway.com/support/tickets/1423150 for more information.

### CODE REVIEW

File: docs/loadbalancer-annotations.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ The default is the first zone of the cluster's region.
 This is the annotation to set the time between two consecutive health checks.
 The default value is `5s`. The duration are go's time.Duration (ex: `1s`, `2m`, `4h`, ...).
 
+### `service.beta.kubernetes.io/scw-loadbalancer-health-check-send-proxy`
+This is the annotation to set if the health checks send a proxy protocol v2 frame before the actual check.
```
Comment (Nox-404) [inline docs/loadbalancer-annotations.md:57]:
```suggestion
This is the annotation to control if proxy protocol should be activated for the health check.
```

Comment (Nox-404) [reply docs/loadbalancer-annotations.md:57]: to match the official documentation

File: scaleway/loadbalancers_annotations.go [hunk 2]
```diff
@@ -44,6 +44,10 @@ const (
 	// The default value is "5s". The duration are go's time.Duration (ex: "1s", "2m", "4h", ...)
 	serviceAnnotationLoadBalancerHealthCheckDelay = "service.beta.kubernetes.io/scw-loadbalancer-health-check-delay"
 
+	// serviceAnnotationLoadBalancerHealthCheckSendProxy is the annotation to control if health check should send proxy protocol message
```
Comment (Nox-404) [inline scaleway/loadbalancers_annotations.go:47]:
```suggestion
	// serviceAnnotationLoadBalancerHealthCheckSendProxy is the annotation to control if proxy protocol should be activated for the health check.
```

**Comment by erebe** [comment]
Thanks, updated :)

**Review by Nox-404 — APPROVED** [review]
thanks for your contribution !

**Pull request merged**